tables_app/src/main/java/org/opendatakit/tables/fragments/TableManagerFragment.java

```diff
@@ -228,7 +231,7 @@ public boolean onContextItemSelected(MenuItem item) {
 
     if (itemId == R.id.table_manager_delete_table) {
       AlertDialog confirmDeleteAlert;
-      AlertDialog.Builder alert = new AlertDialog.Builder(this.getActivity());
+      MaterialAlertDialogBuilder alert = new MaterialAlertDialogBuilder(this.getActivity());
```

Comment:
I see here you didn't pass the` OdkXAlertDialogStyle` as a parameter. 

passing a style is not strictly necessary, but it allows you to customize the appearance of the dialog. When you pass `R.style.OdkXAlertDialogStyle `as a parameter to the` MaterialAlertDialogBuilder,` it applies the styles defined in `OdkXAlertDialogStyle` ( Android Library ) to the dialog. This can include changes to colors, text sizes, button styles, and more, allowing for a consistent design language throughout the app.

Reply:
If you don’t pass a style, the dialog will inherit the default `Material Design style `defined in the app theme. However, if you want to ensure that the dialog matches a specific design or branding, passing a custom style is the way to go. It gives you more control over the look and feel of the dialog.